controllers/cloudstackcluster_controller.go

```diff
@@ -69,52 +71,58 @@ func (r *CloudStackClusterReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 	// Get CloudStack cluster.
 	csCluster := &infrav1.CloudStackCluster{}
-	if retErr = r.Client.Get(ctx, req.NamespacedName, csCluster); retErr != nil {
-		if client.IgnoreNotFound(retErr) == nil {
+	if err := r.Client.Get(ctx, req.NamespacedName, csCluster); err != nil {
+		if client.IgnoreNotFound(err) == nil {
 			log.Info("Cluster not found.")
 		}
-		return ctrl.Result{}, client.IgnoreNotFound(retErr)
+		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
 	// Get CAPI cluster.
-	cluster, retErr := util.GetOwnerCluster(ctx, r.Client, csCluster.ObjectMeta)
-	if retErr != nil {
-		return reconcile.Result{}, retErr
-	} else if cluster == nil {
+	capiCluster, err := util.GetOwnerCluster(ctx, r.Client, csCluster.ObjectMeta)
```

Comment:
I've decided I like the naming scheme of cs for cloudstack objects, and capi for capi objects.

In this case it clarifies quite a bit. Instead of `csCluster` & `cluster` we have `csCluster` & capiCluster`.